packages/ui-app/src/util/isValidBalanceExpectedResponses.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { IsValidWithMessage } from './types';
+
+export enum ErrorMessage {
+  BalanceMustContainOnlySingleEOrPlus = 'Scientific notation may only contain one instance of \'e\' for scientific notation or exponential with \'e+\'',
```

Comment:
No. All messages needs to be translatable when they are used on the UI. As it stand, these are not.

Reply:
I'm only using `ErrorMessage` as a helper for unit testing purposes in isValidBalance.spec.js,  but not on the UI